Sources/Logger/LogManager.swift

```diff
@@ -46,15 +47,21 @@ public class LogManager {
 
     public var loggingConcurrencyMode: LoggingConcurrencyMode = .asyncSerial
 
-    private let serialLoggingQueue = DispatchQueue(label: Constants.Queues.serial, qos: .background)
-    private let concurrentLoggingQueue = DispatchQueue(label: Constants.Queues.concurrent, qos: .background, attributes: .concurrent)
+    let serialLoggingQueue: DispatchQueue
+    let concurrentLoggingQueue: DispatchQueue
 
-    private var loggers: [Logging]
+    var loggers: [Logging]
 
     private let applicationCallbackLogger = ApplicationCallbackLogger()
     private let metaInformationLogger = MetaInformationLogger()
 
-    private init() {
+    init(
```

Comment:
Preferoval bych init, pouze s jednou serialQueue a to s tou ktera se realne bude pouzivat. Za me bych loggingConcurrencyMode dal primo do kontruktoru a neumoznil zmenu za behu loggeru

Reply:
Dobrej point, nakonec predelam celou praci s tema queues.